app-productivity/libreoffice/autobuild/build

```diff
@@ -96,14 +96,13 @@ abinfo "Autogen-ing ..."
              --disable-dependency-tracking \
              --enable-pch=full
 
-if ! ab_match_arch powerpc && \
-   ! ab_match_arch ppc64 && \
-   ! ab_match_arch loongson3; then
+if ab_match_arch loongarch64 || \
+   ab_match_arch riscv64; then
```

Comment:
Missing amd64/arm64?

Reply:
The patch is for loongarch64 only, but is firebird required on amd64/arm64?

Reply:
This is for preloading the source code to patch.

Firebird is required everywhere, but this procedure is for patching the source.

On other architectures its source code will be downloaded automatically.